CORE-1939 Add usage percentage to Data nav icon

psarando opened the pull request:
This PR will add a badge to the Data nav icon, with the user's Data usage percentage in a tooltip, when the usage is 50% or higher.

The badge will be an info icon for usage percentages at or above 50%, a warning icon for usage at or above 75%, and an error icon for usage at or above 100%.

![50 percent usage](https://github.com/cyverse-de/sonora/assets/996408/6555fc4d-edf1-4e9c-8cf2-93193bcef52a)
![50 percent usage with tooltip](https://github.com/cyverse-de/sonora/assets/996408/eb05b5eb-42dd-47cd-9077-246c17baaea5)
---
![75 percent usage](https://github.com/cyverse-de/sonora/assets/996408/9031f284-ca4f-465e-8268-5f2391a2993e)
![75 percent usage with tooltip](https://github.com/cyverse-de/sonora/assets/996408/ed82492d-9dbb-4a13-8b74-14a1473e940d)
---
![100 percent usage](https://github.com/cyverse-de/sonora/assets/996408/ff64c056-5ff8-46df-8b93-80cc6875ca3a)
![100 percent usage with tooltip](https://github.com/cyverse-de/sonora/assets/996408/e95de8a9-b80d-46b3-bfea-ca3ae27aafc7)

slr71 commented on src/components/dashboard/dashboardItem/CPUConsumption.js:
```diff
@@ -64,7 +65,7 @@ const options = (usage, quota, timestamp, title, theme, t) => {
                     title,
                     t("consumptionChartSecondaryTitle", {
-                        percentage: ((usage / quota) * 100).toFixed(2),
+                        percentage: formatUsagePercentage(usage, quota),
```
👍

slr71 reviewed (approved):
Looks good to me! :shipit:

psarando commented:
I updated the warning icon (for usage 75 ~ 100%) to a yellow color from our theme's palette, and I added screenshots to the PR that show the tooltips for each icon.

I'll go ahead and merge this now, but additional reviews still welcome 🙇
